File: tests/rosetta/transpiler/php/call-a-foreign-language-function.php
<?php
ini_set('memory_limit', '-1');
function strdup($s) {
  global $main;
  return $s . '';
}
function main() {
  global $strdup;
  $go1 = 'hello C';
  $c2 = strdup($go1);
  echo rtrim($c2), PHP_EOL;
}
main();

File: tests/rosetta/transpiler/Java/call-a-foreign-language-function.java
public class Main {

    static String strdup(String s) {
        return s + "";
    }

    static void main() {
        String go1 = "hello C";
        String c2 = String.valueOf(strdup(go1));
        System.out.println(c2);
    }
    public static void main(String[] args) {
        main();
    }
}
